components/List/CategoriesList.vue

```diff
@@ -31,6 +31,12 @@ export default {
         ...mapState('list', {
             elements: state => state.elements,
         }),
+        elementsByGroupInLanguage() {
```

Comment:
elementsByGroupInLanguage doesn't fit - computed is returning elements by language not by group